.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "Realm/ObjectStore"]
 	path = Realm/ObjectStore
 	url = https://github.com/realm/realm-object-store.git
-	branch = master
+	branch = jg-handover
```

Comment:
Yes, I plan to update the PR once https://github.com/realm/realm-object-store/pull/116 is merged.